src/extractors/withTranslationHOC.ts

```diff
@@ -214,7 +214,10 @@ function findTFunctionCallsFromPropsAssignment(
   // function
   const tCalls = Array<BabelCore.NodePath<BabelTypes.CallExpression>>();
   for (const tCall of tReferences) {
-    if (tCall.parentPath.isCallExpression()) {
+    if (
+      tCall.parentPath.isCallExpression() &&
```

Comment:
You can keep the `isCallee` function as it was before except that you pass to it `path.parentPath` (instead of taking the parentPath in the function body). It would fix the typing error and make things more easy to follow IMO.

Reply:
OK, I've tried something else.

Reply:
I missed that `isCallee` had to type guard the parent path and not the node path, this is my bad. Sorry to consume more of your precious time on this, but I reckon you can still get rid of the `parent` argument by using a type guard of this form:

```typescript
function isCallee(
  path: BabelCore.NodePath,
): path is BabelCore.NodePath & {parentPath: BabelCore.NodePath<BabelTypes.CallExpression>} {
  return path.parentPath.isCallExpression() && path === path.parentPath.get('callee');
}
```

Reply:
Yep, I think it works!